src/main/java/com/ace/alfox/AlfoxApplication.java

```diff
@@ -5,11 +5,10 @@
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.web.servlet.config.annotation.CorsRegistry;
 import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry;
 import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
 
-@EnableWebMvc
 @SpringBootApplication
-@EnableAutoConfiguration
```

Comment:
This was hell to find. Apparently you can't have both `@EnableWebMvc` and `@SpringBootApplication` or it clears out the default configuration and won't serve any static files.

Reply:
Yeahhhhhh